README.md

```diff
@@ -1,5 +1,30 @@
 # Onigumo #
 
+## About
+
+Onigumo slouží k procházení webových aplikací či statických webových stránek.
+Obsah a případně další příslušná metatdata konkrétní *url* adresy uloží do
+strukturované podoby dat (např. JSON nebo YAML), které je vhodné
```

Comment:
Ono nakonec bude třeba se rozhodnout, jestli JSON, nebo YAML. Případně přidat přepínač. Ale tady je to možná zcela nadbytečné.